core-java/src/test/java/com/baeldung/java/listInitialization/ListInitializationUnitTest.java

```diff
@@ -0,0 +1,74 @@
+package com.baeldung.java.listInitialization;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ListInitializationUnitTest {
+	
+	@Rule
+	public ExpectedException exception = ExpectedException.none();
+
+	@Test
+	public void listInOneLine1(){
```

Comment:
@Vizsoro Just a quick reminder to use the BDD naming convention as described in the docs.

Reply:
Done.